test/functional/tests/install/test_install_rpm.py

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+from time import sleep
+
+import pytest
+
+from api.cas import casadm
+from core.test_run import TestRun
+
+
+@pytest.mark.remote_only()
+@pytest.mark.without_install()
+def test_load_x_to_one_without_params():
+    """
+        title: Test for installing OpenCAS from RPM
+        description: |
+          Prepare SRPM for OpenCAS, build OpenCAS's RPM from SRPM,
+          install OpenCAS from RPM and then uninstall OpenCAS.
+        pass_criteria:
+          - OpenCAS installs successfully
+          - OpenCAS uninstalls successfully
+    """
+    with TestRun.step("Check package manager."):
+        install_command = TestRun.executor.check_install_command()
+
+    with TestRun.step("Create OpenCAS's SRPM."):
+        TestRun.executor.run_expect_success(f"make srpm -C {TestRun.usr.working_dir}")
+
+    with TestRun.step("Create OpenCAS's RPM."):
+        output = TestRun.executor.run_expect_success(
+            f"make rpm -C {TestRun.usr.working_dir}").stdout.splitlines()
+        interesting_output = []
+        for line in output:
+            if "RPMS/x86_64" in line:
+                interesting_output.append(line)
+            elif "You can find your fresh packages in" in line:
+                interesting_output.append(line)
+        program = interesting_output[0].split("/")[-1]
+        modules = interesting_output[1].split("/")[-1]
+        path = interesting_output[2].split("'")[1]
+
+    with TestRun.step("Install OpenCAS from RPM."):
+        TestRun.executor.run(f"{install_command} {path}/{modules} -y")
+        TestRun.executor.run(f"{install_command} {path}/{program} -y")
+
+    with TestRun.step("Check if OpenCAS is installed correctly."):
+        sleep(1)
+        try:
+            casadm.print_version()
+            TestRun.LOGGER.info("OpenCAS is installed as expected.")
+        except Exception:
+            TestRun.fail("OpenCAS should be installed.")
+
+    with TestRun.step("Uninstall OpenCAS from RPM."):
+        TestRun.executor.run(f"{install_command.split()[0]} remove open-cas-linux* -y")
```

Comment:
It looks like the install command needs to be guessed by framework, but the remove command can be hardcoded :)
Again, using plain `rpm` solves the problem. I wouldn't expect RPM to be installable on the system, where `rpm` tool is not present anyway.